config/routes.rb

```diff
@@ -5,6 +5,9 @@
   # Defines the root path route ("/")
   # root "articles#index"
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index show new create] do
+      resources :comments, only: %i[create]
```

Comment:
- [ ] There should be a route for the `new` action which should render a form in the `views/comments/new.html.erb` file which accepts input for creating comments.

Reply:
Dear @codepantha 
I think this refers to the above inquire as the action controller can't be include yet its not defined. Thank you.

Reply:
@AlphaNtihinduka I have addressed this above ☝️